routes/university/universityPublic.js

```diff
@@ -40,7 +40,7 @@ router.get('/unapproved', async (req, res) => {
 // Get all university information by _id
```

Comment:
Comment doesn't match

Reply:
actually, just did a search of the frontend, and this endpoint isn't used at all anyways